src/itest/kotlin/nl/lifely/zac/itest/SmartDocumentsTest.kt

```diff
@@ -62,51 +64,51 @@ class SmartDocumentsTest : BehaviorSpec({
             }
         }
 
-        When("the create mapping endpoint is called") {
+        When("the create mapping endpoint is called with correct payload") {
             val smartDocumentsZaakafhandelParametersUrl =
                 "$ZAC_API_URI/smartdocuments/templates/zaakafhandelParamaters/$ZAAKTYPE_MELDING_KLEIN_EVENEMENT_UUID"
             val restTemplateGroups = """
                 [
                   {
-                    "id": "6e0658a4-3039-4a7c-84d2-4ef47285ea52",
+                    "id": "D5037631FF6748269059B353699EFA0C",
```

Comment:
Why not create constants out of these 'UUID's? Some of them are used more than once so I think it is more readable & maintainable to make constants out of them and put them in `ItestConfiguration`?

Reply:
Moved the IDs and names to ITestConfiguration